coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/Calcite2TrinoUDFConverter.java

```diff
@@ -160,6 +173,13 @@ public RexNode visitCall(RexCall call) {
         return GenericProjectToTrinoConverter.convertGenericProject(rexBuilder, call);
       }
 
+      if (call.getOperator().getName().equals("from_utc_timestamp")) {
+        Optional<RexNode> modifiedCall = visitFromUtcTimestampCall(call);
+        if (modifiedCall.isPresent()) {
+          return modifiedCall.get();
```

Comment:
Was it possible to handle this with a `UDFTransformer`?

Reply:
No, it is very limited right now. It feels extending it would be painful. Basically what we would need to do is to allow expressing logic I have here coded in Java in JSON we pass, to setup parameter translation. I strongly belive java code is better tool for the job.